openapi/p4pa-debt-position.openapi.yaml

```
@@ -789,6 +789,46 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/DebtPositionErrorDTO'
+  /debt-positions/{debtPositionId}/delete:
```

is redundant to have the action on the path, we have already the HTTP method to express the delete operation
```suggestion
  /debt-positions/{debtPositionId}:
```